packages/webview_flutter_lwe/tizen/src/webview.cc

```diff
@@ -212,9 +226,11 @@ void WebView::RegisterJavaScriptChannelName(const std::string& name) {
         {flutter::EncodableValue("channel"), flutter::EncodableValue(name)},
         {flutter::EncodableValue("message"), flutter::EncodableValue(message)},
     };
-    webview_channel_->InvokeMethod(
-        "javaScriptChannelMessage",
-        std::make_unique<flutter::EncodableValue>(args));
+    dispatcher->dispatchTaskOnMainThread([&]() {
```

Comment:
How about narrowing the scope of the capture rather than using `&`? You could use `[this, &args]`.